build_system.sh

```diff
@@ -86,22 +113,18 @@ DATETIME=$(date '+%Y-%m-%dT%H:%M:%S')
 GIT_HASH=$(git --git-dir=$DIR/.git rev-parse HEAD)
 docker exec -w $ROOTFS_DIR $MOUNT_CONTAINER_ID bash -c "printf \"$GIT_HASH\n$DATETIME\" > BUILD"
 
-# Unmount image
-echo "Unmount filesystem"
-docker exec $MOUNT_CONTAINER_ID umount -l $ROOTFS_DIR
-docker rm -f $MOUNT_CONTAINER_ID > /dev/null
-
-cd $DIR
-
 # Sparsify
-echo "Sparsify image"
-TMP_SPARSE="$(mktemp)"
-img2simg $ROOTFS_IMAGE $TMP_SPARSE
-mv $TMP_SPARSE $SPARSE_IMAGE
+echo "Sparsify image $(basename $SPARSE_IMAGE)"
```

Comment:
I think we can just remove this https://github.com/commaai/agnos-builder/issues/5. want to make a separate PR to remove it and confirm it won't break anything?

IIRC we kept it around since the very first openpilot versions for C3 only worked with sparse images

Reply:
I wanted to suggest this also. I think `agnos-builder` (fastboot and edl), `comma flash` (fastboot and edl) and `openpilot updater` will work fine with the normal image. I don't see why not, we need to test.

When you say remove https://github.com/commaai/agnos-builder/issues/5 you mean remove both sparsified images, right?

Reply:
Actually, `system.img.raw` is 10GB, sp we need one sparsified image. I think we can just remove `system-skip-chunks.img`, since fastboot and edl supports both types of sparsified images and openpilot updater I think it uses the non-"skip-chunks" one.

Reply:
raw agnos is ~10GB so I think we still want sparse image for faster download. skip-chunk-img will speed up flash speed for fastboot since it would just skip and go to the next flash location. There is no difference in speed for QDL or even speed it up, QDL populates the data for the skip chunk area, removing it might saves some flashing time

Reply:
let's continue this discussion in https://github.com/commaai/agnos-builder/issues/5 or https://github.com/commaai/agnos-builder/pull/275